raiden/tests/scenarios/scenario-funding channel.yaml

```diff
@@ -0,0 +1,256 @@
+settings:
+  #notify: "raiden-scenario-basic@example.com"
+  ## either `fast`, `medium` or an integer (in wei)
+  gas_price: 100_000_000_000
+
+token:
+  ## To re-use an existing token give an address
+  #address: "0x9CBA8b93C3A61F9417332B0e58E613f0F1168C2c"
+
+  ## To deploy a new one remove the `address` key
+  ## Optionally parameters:give name and symbol, both will be autogenerated if not given
+  ## name - Token name, default: autogenerated
+  ## symbol - Token symbol, default: autogenerated
+  ## decimals - Token decimals, default: 0
+  ## balance_min - Minimum node balance, default: 5000
+  name: "Domi2004"
```

Comment:
Would be nice to remove this, as we filter out the default names at the explorer but this name will go through.